src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,156 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> firstNode;
+    private Node<T> lastNode;
+    private int size;
+
+    public MyLinkedList() {
+    }
+
     @Override
     public boolean add(T value) {
-        return false;
+        addLastValue(value);
+        return true;
     }
 
     @Override
     public void add(T value, int index) {
-
+        if (index == size) {
+            addLastValue(value);
+        } else {
+            addFirstValue(value, findNodeByIndex(index));
```

Comment:
addFirstValue is not quite correct naming since, you don't only use this method to add value to first node